lib/advisor/core/answers/README.md

```diff
@@ -0,0 +1,19 @@
+# Answers
+
+Anything related to storing and finding answers.
+Normally, one would look up the ansers to a given [advice](lib/advisor/core/advice/README.md) entity. This module can also look through an entire list of advice entities.
```

Comment:
Typo: "answers"

Am I correct in thinking that answers is pretty tied to advices? Feels a bit odd that this module can do various searches through advices.